### PR TITLE
Add Addiction_3D to release models

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: RealFlight/Released_Models/QuadPlanes/Tailsitters/Addiction_3D/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Aircraft Model:
+3D Model By: Erik Peterson - AKA: Keylitho - 
+
+
+![JPG](https://github.com/ArduPilot/SITL_Models/raw/master/RealFlight/Released_Models/QuadPlanes/Tailsitters/Addiction_3D/Addiction.jpg)
+
+Model modifications by : Andrew Tridgell
+
+
+* Wing Span....... 23 (in)
```
Comment (kd0aij) [inline RealFlight/Released_Models/QuadPlanes/Tailsitters/Addiction_3D/README.md:12]: the RealFlight physics for this model has half-span of 585mm which equates to a full span of 46"
But the aileron length is 640mm ???  probably should be 580mm

**Review by kd0aij — CHANGES_REQUESTED** [review]
Very little roll/pitch authority in QHOVER. Is this the best it can do?
FBWA seems good
ACRO with yaw rate at 60 deg/sec seems really weak on yaw authority compared to FBWA

**Comment by kd0aij** [comment]
Tune seems good in QHOVER, FBWA and ACRO. I'm assuming my frame rate is too low for QACRO though since I get severe roll oscillations.

**Comment by Hwurzburg** [comment]
flies okay for me...whats your frame rate?

**Comment by Hwurzburg** [comment]
@kd0aij thanks for testing and helping improve....

**Pull request merged**